### PR TITLE
fix: responsive design dashboard

### PR DESCRIPTION
**Pull request by Naiby**

### CODE REVIEW

File: src/enums/TypeButton.enum.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+  'primary' = 'primary',
+  'secondary' = 'secondary',
+  'accent' = 'accent',
+  'ghost' = 'ghost',
+  'link' = 'link',
+  'info' = 'info',
+  'success' = 'success',
+  'warning' = 'warning',
+  'error' = 'error',
+  'null' = '',
```
Comment (AlefrankM) [inline src/enums/TypeButton.enum.tsx:11]: Please get rid off this

File: src/pages/login.tsx [hunk 2]
```diff
@@ -35,12 +36,12 @@ export default function LoginPage() {
                 active={false}
                 aria-label="Close"
                 circle={false}
-                color="primary"
                 disabled={false}
                 extra=""
                 outline={false}
                 size="lg"
                 square={false}
+                style={TypeButton.primary}
```
[the remark this answers is not in the document]
Comment (juliswer) [reply src/pages/login.tsx:44]: I agree. Maybe "variant" might be a better approach

**Review by juliswer — COMMENTED** [review]
Looks good. Please check my comments

File: src/features/app/actions/interview/saveInterview/saveInterview.ts [hunk 3]
```diff
@@ -1,24 +1,19 @@
...
+import { openToastSuccess } from '@/store/ui/ui.slice'
 import { createAsyncThunk } from '@reduxjs/toolkit'
 
 export const saveInterview = createAsyncThunk(
   'app/saveInterview',
   async (interview: Interview, { dispatch }) => {
-    try {
-      const cStack = createCommandStack<Interview>(interview)
+    const cStack = createCommandStack<Interview>(interview)
+    console.log(interview)
```
Comment (juliswer) [inline src/features/app/actions/interview/saveInterview/saveInterview.ts]: remove clg

**Review by juliswer — APPROVED** [review]
Good Job!

**Pull request merged**